### PR TITLE
[Docs]: Add ops definition 

### PR DESCRIPTION
**Pull request by RunningLeon**
## Motivation

This PR adds a document about custom operators' definition in `mmcv` for pytorch2onnx. It is intended to make the ops definition standard and popular in the community. Current following ops are included in this document:

- [x] MMCVBorderAlign
- [x] MMCVCARAFE
- [x] MMCVCAWeight
- [x] MMCVCAMap
- [x] MMCVCornerPool
- [x] MMCVDeformConv2d
- [x] MMCVModulatedDeformConv2d
- [x] MMCVDeformRoIPool
- [x] MMCVMaskedConv2d
- [x] MMCVPSAMask
- [x] NonMaxSuppression
- [x] MMCVRoIAlign
- [x] MMCVRoIAlignRotated

### torch ops

There are some ops from PyTorch that are not supported when exporting to ONNX. They are included in here as well.

- [x]  cummax
- [x]  cummin

## Modification

None

## BC-breaking (Optional)

No

### CODE REVIEW

File: docs/deployment/mmcv_ops_definition.md [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+<dl>
+<dt><tt>output</tt>: T</dt>
+<dd>The upsampled features. 4-D tensor of shape (N, C, H * scale_factor, W * scale_factor). N is the batch size.</dd>
+</dl>
+
+### Type Constraints
+
+- T:tensor(float32)
+
+## MMCVCAWeight
```
[the remark this answers is not in the document]
Comment (RunningLeon) [reply docs/deployment/mmcv_ops_definition.md:183]: OK

**Comment by RunningLeon** [comment]
@zhouzaida Any comments on this PR?

**Comment by zhouzaida** [comment]
The definitions of some ops are repeated with other files like https://github.com/open-mmlab/mmcv/blob/master/docs/deployment/tensorrt_custom_ops.md. Is it necessary to delete other repeated files?

**Comment by RunningLeon** [comment]
> The definitions of some ops are repeated with other files like https://github.com/open-mmlab/mmcv/blob/master/docs/deployment/tensorrt_custom_ops.md. Is it necessary to delete other repeated files?

May keep them as references and remove them later when everything becomes clear and stable.

**Review by zhouzaida — APPROVED** [review]

**Review by grimoire — APPROVED** [review]

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**